Build error on `android/termux`

```
[github.com/projectdiscovery/ratelimit](http://github.com/projectdiscovery/ratelimit)
# [github.com/projectdiscovery/utils/permission](http://github.com/projectdiscovery/utils/permission)
go/pkg/mod/github.com/projectdiscovery/utils@v0.0.58/permission/permission_other.go:10:6: checkCurrentUserRoot redeclared in this block
        go/pkg/mod/github.com/projectdiscovery/utils@v0.0.58/permission/permission_linux.go:16:6: other declaration of checkCurrentUserRoot
go/pkg/mod/github.com/projectdiscovery/utils@v0.0.58/permission/permission_other.go:15:6: checkCurrentUserCapNetRaw redeclared in this block
        go/pkg/mod/github.com/projectdiscovery/utils@v0.0.58/permission/permission_linux.go:21:6: other declaration of checkCurrentUserCapNetRaw
[github.com/projectdiscovery/mapcidr](http://github.com/projectdiscovery/mapcidr)
```

ref: https://github.com/projectdiscovery/httpx/pull/1424

--- a/go.mod
+++ b/go.mod
@@ -34,7 +34,7 @@ require (
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/projectdiscovery/blackrock v0.0.1 // indirect
-	github.com/projectdiscovery/utils v0.0.60
+	github.com/projectdiscovery/utils v0.0.61
 	github.com/saintfish/chardet v0.0.0-20230101081208-5e3ef4b5456d // indirect
 	go.uber.org/multierr v1.11.0 // indirect
 	golang.org/x/net v0.17.0 // indirect
@@ -72,7 +72,7 @@ require (
 	github.com/mattn/go-isatty v0.0.19 // indirect
 	github.com/mattn/go-runewidth v0.0.14 // indirect
 	github.com/mholt/archiver v3.1.1+incompatible // indirect
-	github.com/minio/selfupdate v0.6.0 // indirect
+	github.com/minio/selfupdate v0.6.1-0.20230907112617-f11e74f84ca7 // indirect
 	github.com/muesli/reflow v0.3.0 // indirect
 	github.com/muesli/termenv v0.15.1 // indirect
 	github.com/nwaples/rardecode v1.1.3 // indirect

